login.html

```diff
@@ -1,11 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="login.css">
     <title>Welcome to the postIt! Please Log In</title>
 </head>
+
 <body>
+    <section class="wrapper">
+        <div class="content">
+
+            <header>
+                <div class="image-container">
+                    <img src="http://assets.stickpng.com/thumbs/5b06c181fad1cae04539afdd.png">
```

Comment:
I get a 403 on this for some reason, could replace it with an asset?

Reply:
Will add the picture as an asset and link it